opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/InteroperabilityTest.java

```diff
@@ -335,6 +357,151 @@ public void testNoRecordDoesNotExport() {
     verify(spanExporter, never()).export(anyCollection());
   }
 
+  @Test
+  @SuppressWarnings("deprecation") // Summary is deprecated in census
+  void testSupportedMetricsExportedCorrectly() {
```

Comment:
seems like some non-gradle work crept into this PR. Can we limit it to just gradle changes?

Reply:
Yup moved